_pytest/config.py

```diff
@@ -942,6 +941,7 @@ def _consider_importhook(self, args, entrypoint_name):
         and find all the installed plugins to mark them for re-writing
         by the importhook.
         """
+        import pkg_resources
         ns, unknown_args = self._parser.parse_known_and_unknown_args(args)
         mode = ns.assertmode
         if mode == 'rewrite':
```

Comment:
Sorry, I haven't been able to properly test this on my code yet, but you are right, the import needs to be moved to the else clause, this change won't be enough. I'll do that. I'd like to help with the test as well, but I think I will need your help :) 

About `assert=plain` I didn't know about it! I'll modify my code to use it. Thanks!